autoware_control_msgs/README.md

```diff
@@ -0,0 +1,13 @@
+# autoware_control_msgs
+
+## Design
+
+Vehicle dimensions and axes: <https://autowarefoundation.github.io/autoware-documentation/main/design/autoware-interfaces/components/vehicle-dimensions/>
+
+### Lateral.msg
+
+### Longitudinal.msg
+
+### Control.msg
+
+### ControlHorizon.msg
```

Comment:
It would be better to summarize the discussions in this PR, and write brief rationales here.

Reply:
Added in https://github.com/autowarefoundation/autoware_msgs/pull/11/commits/35dbbe88cce34d669a91f9ff8f54a560eb48527b